### ISSUE TITLE
Explore better relative URL detection pattern

### ISSUE THREAD

**Issue by dlongley** [digitalbazaar/http-signature-zcap-verify#36]
See originally:

This would disallow other protocols, but I think that's largely untested anyway. I'll file an issue to add support if needed for that.

_Originally posted by @dlongley in https://github.com/digitalbazaar/http-signature-zcap-verify/pull/35#discussion_r1548018222_

**Comment by dmitrizagidulin** [comment]
Running into this as well. Specifically, https://github.com/digitalbazaar/http-signature-zcap-verify/blob/main/lib/index.js#L198 is essentially requiring the use of `https` protocol.
Which makes unit testing on localhost tough.

Would you be open to a PR that opened this up to `http` localhost?

**Comment by dlongley** [comment]
Yeah, that code path could probably be more clever in how it detects relative vs. absolute URLs.

**Comment by dlongley** [comment]
This has been addressed by PR #40. It was released in v11.2. cc: @gobengo
